bin/improver-temp-lapse-rate

```diff
@@ -94,14 +101,22 @@ def main():
     orography_cube = load_cube(args.orography_filepath)
     land_sea_mask_cube = load_cube(args.land_sea_mask_filepath)
 
-    result = LapseRate(
-        max_height_diff=args.max_height_diff,
-        nbhood_radius=args.nbhood_radius,
-        max_lapse_rate=args.max_lapse_rate,
-        min_lapse_rate=args.min_lapse_rate).process(temperature_cube,
-                                                    orography_cube,
-                                                    land_sea_mask_cube)
-
+    if args.return_dalr:
+        result = temperature_cube.copy(
+            data=np.full_like(temperature_cube.data, U_DALR.points[0]))
+        result.rename('lapse_rate')
+        result.units = U_DALR.units
+        for attr in ['title', 'source', 'history', 'um_version']:
+            if attr in result.attributes.keys():
+                result.attributes.pop(attr)
```

Comment:
This results in different attributes on the output cube when this flag is set, as can be seen in the basic and dalr kgo files. If popping these attributes is necessary, then this should also be done in the plugin.
Ideally, the cube meta-data generation should be refactored into a function of the plugin so that we can be confident of getting the same meta-data on the cube whether this switch is set or not.

Reply:
In discussion with Caroline and Gavin, we agreed to move the meta-data changes into a function that is called by the plugin and CLI so that the output are consistent.

Reply:
I've used the existing `amend_metadata` function here and applied this to the output from either path for calculating the lapse rate, so the attributes present on the output should be consistent.